tests/dragonfly/seeder/__init__.py

```diff
@@ -79,13 +79,15 @@ def __init__(
         data_size=100,
         variance=5,
         samples=10,
+        element_size_ratio=1 / 3,
```

Comment:
can we instead use element count?
When writing a tests case I want to define the total size of the datastructor and the number of elements it will have , each element size will be total size / element count
I find element count param much more intuitive than element size ratio which defines the element size which will be total size ^   element size ratio